src/web/mod.rs

```diff
@@ -547,16 +513,4 @@ mod test {
             Ok(())
         });
     }
-
-    #[test]
-    fn test_latest_version() {
-        let versions = vec!["1.0.0".to_string(),
-                            "1.1.0".to_string(),
-                            "0.9.0".to_string(),
-                            "0.9.1".to_string()];
-        assert_eq!(latest_version(&versions, "1.1.0"), None);
-        assert_eq!(latest_version(&versions, "1.0.0"), Some("1.1.0".to_owned()));
-        assert_eq!(latest_version(&versions, "0.9.0"), Some("1.1.0".to_owned()));
-        assert_eq!(latest_version(&versions, "invalidversion"), None);
```

Comment:
Please add these test cases to the new `test_latest_version` function. It would also be great if you could add a test case that these are sorted by semver instead of alphabetically: `1.12.0` comes lexicographically before `1.2.0`, but should come after when sorted by version.

Reply:
Good idea! I have added these edge-cases to `test_releases_should_be_sorted` since this test focuses solely on `releases` being sorted correctly.
I would prefer not to put the same test cases in `test_latest_version` to avoid having duplicate tests.

Is that acceptable for you?

Reply:
Looks good to me :)